src/main/java/gui/MenuController.java

```diff
@@ -346,7 +364,12 @@ private void bookmarked(MenuItem bookmark) {
      * @return The sequenceMap.
      */
     HTreeMap<Long, String> getSequenceHashMap() {
-        return DrawableCanvas.getInstance().getParser().getSequenceHashMap();
```

Comment:
Shouldn't the calls to this method be in a try/catch instead of the contents of the method?